### ISSUE TITLE
Add ability to select channel

### ISSUE THREAD

**Issue by ghost** [Malopieds/InnerTune#163]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Google lets users create channels with other names on their personal accounts. When logging in InnerTune there is no option to choose any of them, instead app uses personal account. But in original YT Music you can choose channel instead of personal account.

### Why do you want this feature?

I'm using channel instead of personal account for 5+ years (because I don't want to upload videos on channel with my real name lol) and all of my music is here.

**Comment by mikooomich** [comment]
You can use your channel (I'm doing so right now), as long as you get to the YouTube music page somehow. If you land on the regular YouTube page after logging in, under accounts, click on YouTube music. If that isn't there, what I did was put a clickable link on my channel about section (the login screen is just an ordinary browser).

**Comment by tadly** [comment]
Doesn't work for me.
I did switch the account inside the "Login" page for both youtube and youtube music but when I then return to the app, the playlists shown are still the ones from the main account, not the channel.
I've also restarted the app (swiped from recents and force stopped) which didn't help either.

**Comment by conordy** [comment]
yep, doesn't work for me either, it just selects channel on web page, but the app still uses personal account
